core/src/main/java/org/ethereum/beacon/core/BeaconBlock.java

```diff
@@ -72,4 +75,18 @@ public Bytes96 getSignature() {
   public BeaconBlockBody getBody() {
     return body;
   }
+
```

Comment:
Just an idea to simplify equals: `getHash().equals(o.getHash())` ?
I would also add `hashCode()` whenever you add `equals()` (It can be also derived from `getHash()`)

Reply:
I wanted to test at least something, but current getHash() is a stub